cpu/stm32f4/startup.c

```diff
@@ -56,12 +56,12 @@ void reset_handler(void)
     uint32_t *src = &_etext;
 
     /* load data section from flash to ram */
-    for (dst = &_srelocate; dst < &_erelocate; ) {
+    for (dst = &_srelocate; dst < &_erelocate;) {
```

Comment:
this and all following changes to the `startup.c` are unrelated to this PR. Please open a separate PR for these.